x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/rest/action/apikey/RestUpdateApiKeyActionTests.java

```diff
@@ -53,6 +53,9 @@ public void testAbsentRoleDescriptorsAndMetadataSetToNull() {
 
         dispatchRequest(builder.build());
 
-        assertEquals(new UpdateApiKeyRequest(apiKeyId, null, null), requestHolder.get());
+        final UpdateApiKeyRequest request = requestHolder.get();
```

Comment:
Removed `equals` from the class (see my other comment) so can't rely on it here anymore.